scripts/install-multi-user.sh

```diff
@@ -97,7 +97,8 @@ is_os_darwin() {
 }
 
 contact_us() {
-    echo "You can open an issue at https://github.com/nixos/nix/issues"
+    echo "You can open an issue at"
+    echo "https://github.com/NixOS/nix/issues/new?labels=installer&template=installer.md"
```

Comment:
Nitpick: is the `labels=intaller` parameter needed since the template already specifies it?

Reply:
Tested this, yes you need it. You get an even longer URL with empy fields from the GitHub UI when filing an issue.